docs/pages/modules/supporting-platforms.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Supporting more platforms
+description: Learn how to add support for macOS and tvOS platforms.
+---
+
+import { Step } from '~/ui/components/Step';
+
+Expo Modules API provides the first-class support for Android and iOS, however since all Apple platforms are based on the same foundation and use the same programming language,
+it is possible to target other Apple platforms in the Expo module. At the moment only **macOS** and **tvOS** platforms (also called [Out-of-Tree platforms](https://reactnative.dev/docs/out-of-tree-platforms)) are supported.
+This guide will walk you through adding support for these platforms.
+
+<Step label="1">
+
+### Use the `"apple"` platform in `expo-module.config.json`
```

Comment:
```suggestion
## Use the `"apple"` platform in `expo-module.config.json`
```

Let's use an H2 heading here for the correct heading sequence as per our [writing styleguide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#headings).